multimedia/tvheadend/Makefile

```diff
@@ -28,110 +24,205 @@ include $(INCLUDE_DIR)/nls.mk
-  CONFIGURE_ARGS += --disable-satip_client
+	CONFIGURE_ARGS += --disable-satip_client
 endif
 
 ifeq ($(CONFIG_TVHEADEND_HDHOMERUN_CLIENT),)
-  CONFIGURE_ARGS += --disable-hdhomerun_static
+	CONFIGURE_ARGS += --disable-hdhomerun_static
 else
-  CONFIGURE_ARGS += --enable-hdhomerun_client
+	CONFIGURE_ARGS += --enable-hdhomerun_client
 endif
 
-# Descrambling
-ifeq ($(CONFIG_TVHEADEND_CWC_SUPPORT),)
-  CONFIGURE_ARGS += --disable-cwc
+## Descrambling
+ifeq ($(CONFIG_BUILD_PATENTED),)
+	CONFIGURE_ARGS += --disable-tvhcsa --disable-dvbcsa --disable-constcw --disable-cwc --disable-capmt --disable-cccam
+else
+ifeq ($(CONFIG_TVHEADEND_CSA),)
+	CONFIGURE_ARGS += --disable-tvhcsa --disable-dvbcsa --disable-constcw --disable-cwc --disable-capmt --disable-cccam
+else
+	CONFIGURE_ARGS += --enable-tvhcsa --enable-dvbcsa
+ifeq ($(CONFIG_TVHEADEND_CONSTCW),)
+	CONFIGURE_ARGS += --disable-constcw
 endif
-
-ifeq ($(CONFIG_TVHEADEND_CAPMT_SUPPORT),)
-  CONFIGURE_ARGS += --disable-capmt
+ifeq ($(CONFIG_TVHEADEND_NEWCAMD),)
+	CONFIGURE_ARGS += --disable-cwc
 endif
-
-ifeq ($(CONFIG_TVHEADEND_CCW_SUPPORT),)
-  CONFIGURE_ARGS += --disable-constcw
+ifeq ($(CONFIG_TVHEADEND_CAPMT),)
+	CONFIGURE_ARGS += --disable-capmt
 endif
-
-# Other options
-ifeq ($(CONFIG_TVHEADEND_AVAHI_SUPPORT),)
-  CONFIGURE_ARGS += --disable-avahi
-else
-  CONFIGURE_ARGS += --enable-avahi
+ifeq ($(CONFIG_TVHEADEND_CCCAM),)
+	CONFIGURE_ARGS += --disable-cccam
 endif
-
-ifeq ($(CONFIG_TVHEADEND_IMAGECACHE),)
-  CONFIGURE_ARGS += --disable-imagecache
-else
-  CONFIGURE_ARGS += --enable-imagecache
 endif
-
-ifeq ($(CONFIG_TVHEADEND_TRACE),)
-  CONFIGURE_ARGS += --disable-trace
 endif
 
-# libav and ffmpeg are broken, so remove codecs too.
+## Transcoding | Uncomment these.
+##ifeq ($(CONFIG_BUILD_PATENTED),)
+##	CONFIGURE_ARGS += --disable-libav --disable-libx264 --disable-libx265 --disable-libx265_static
+##else
+##ifeq ($(CONFIG_TVHEADEND_LIBFFMPEG),)
+##	CONFIGURE_ARGS += --disable-libav
+##else
+##	CONFIGURE_ARGS += --enable-libav
+##endif
+##ifeq ($(CONFIG_TVHEADEND_LIBX264),)
+##	CONFIGURE_ARGS += --disable-libx264
+##endif
+##ifeq ($(CONFIG_TVHEADEND_LIBX265),)
+##	CONFIGURE_ARGS += --disable-libx265 --disable-libx265_static
+##endif
+##ifneq ($(CONFIG_TVHEADEND_LIBFDKAAC),)
+##	CONFIGURE_ARGS += --enable-libfdkaac
+##endif
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBVPX),)
+##	CONFIGURE_ARGS += --disable-libvpx
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBlibtheora),)
+##	CONFIGURE_ARGS += --disable-libtheora
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBVORBIS),)
+##	CONFIGURE_ARGS += --disable-libvorbis
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBOPUS),)
+##	CONFIGURE_ARGS += --disable-libopus
+##endif
+
 CONFIGURE_ARGS += \
 	--arch=$(ARCH) \
+	--disable-libsystemd_daemon \
 	--disable-dbus_1 \
 	--disable-libav \
 	--disable-ffmpeg_static \
-	--disable-libx264 \
-	--disable-libx264_static \
 	--disable-libx265 \
 	--disable-libx265_static \
+	--disable-libx264 \
```

Comment:
I would put this as it was. x264 before x265

Reply:
OK...